src/base/DelegateInputLabelMixin.js

```diff
@@ -31,18 +32,60 @@ export default function DelegateInputLabelMixin(Base) {
       this[setState]({ ariaLabel });
     }
 
+    // Forward ARIA labelledby as an aria-label to the input element.
+    get ariaLabelledby() {
+      return this[state].ariaLabelledby;
+    }
+    set ariaLabelledby(ariaLabelledby) {
+      if (this.getRootNode() !== null) {
+        // @ts-ignore
+        let labelNode = this.getRootNode().querySelector(
+          `#${ariaLabelledby}`
+        );
+
+        labelNode.setAttribute('aria-hidden', true);
+        const ariaLabel = labelNode.innerText;
+        this[setState]({ ariaLabel, ariaLabelledby });
+      }
+    }
+
     get [defaultState]() {
       return Object.assign(super[defaultState] || {}, {
         ariaLabel: "",
+        ariaLabelledby: "",
       });
     }
 
     [render](changed) {
       super[render](changed);
+      if (this[firstRender]) {
+        if (this.getRootNode() !== null) {
+          // @ts-ignore
+          const labelNode = this.getRootNode().querySelector(
+            `[for="${this.id}"]`
+          )
+          if (labelNode !== null) {
+            const ariaLabel = labelNode.innerText;
+            labelNode.setAttribute('aria-hidden', true);
+            this[setState]({ ariaLabel });
```

Comment:
Calling `setState` during rendering is generally something Elix components avoid. That's why this call is producing a console warning.

Recommend moving this logic to a `rendered` callback. That happens after rendering is finished, when it's safe to call `setState` again.